react-components/src/apps/admin/AllStats/AppStatsComponents/JobsTab.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This function consists of everything under the Jobs & Logins tab
+ */
+
+import React from "react";
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+    Paper,
+    TableHead,
+} from "@material-ui/core";
+import ids from "./AllStatsIDs";
+import { withI18N, getMessage, formatMessage, build } from "@cyverse-de/ui-lib";
+import messages from "./messages";
+import { injectIntl } from "react-intl";
+
+function JobsTab(props) {
+    let rows = props.jobs,
+        distinctLoginData = props.distinctLoginData;
+    const { baseId } = props;
+
+    return (
+        <Paper id={build(baseId, ids.PAPER)}>
+            <Table aria-label="simple table">
+                <TableHead id={build(baseId, ids.PAPER, ids.HEADER)}>
+                    <TableRow>
+                        <TableCell> {getMessage("jobType")} </TableCell>
+                        <TableCell align="center">
+                            {getMessage("jobCount")}
+                        </TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody id={build(baseId, ids.PAPER, ids.TABLE)}>
+                    {rows.map((row) => (
+                        <TableRow>
+                            <TableCell>
+                                {getMessage("jobsData", {
+                                    values: {
+                                        category: row.Category,
+                                        jobStatus: row.Status,
+                                    },
+                                })}
+                            </TableCell>
+                            <TableCell align="center">{row.Count}</TableCell>
+                        </TableRow>
+                    ))}
+                    <TableRow>
+                        <TableCell>
+                            {formatMessage(props.intl, "distinctLogins")}
+                        </TableCell>
+                        <TableCell align="center">
+                            {distinctLoginData.count}
+                        </TableCell>
+                    </TableRow>
+                    <TableRow>
+                        <TableCell>
+                            {formatMessage(props.intl, "totalLogins")}
```

Comment:
Just a minor thing, but I don't think `formatMessage` is necessary in a table - `getMessage` should work just like in the TableCell a few rows up.

Reply:
Right, since `getMessage` will return a React component that renders as the message, and `formatMessage` only returns a string, which is required in places that can't render a component, such as `label` properties.

The `getMessage` call looks a bit cleaner and doesn't require the `intl` param, so we generally prefer that where we can use it.

Reply:
Now that `formatMessage` is gone, you can also remove the `formatMessage` and `injectIntl` imports.